npm/test-install.js

```diff
@@ -0,0 +1,18 @@
+const os = require('os');
+const path = require('path');
+const fs = require('fs');
+const installer = require('./install.js');
+const lib = require('./lib');
+const assert = require('assert');
+
+describe('E2E Install', function () {
+    it('should install binary', async function() {
+        this.timeout(5000);
+        await installer.install();
+    
+        let binaryName = lib.getBinaryName(os.type(), os.arch());
+        let dest = path.join(__dirname, `bin/${binaryName}`);
```

Comment:
Should probably pass `bin` as it's own arg to path.join

`let dest = path.join(__dirname, 'bin', binaryName)`

Reply:
Good catch. Updated

Reply:
Looks like you changed the one in install.js but not npm/test-install.js

Reply:
I didn't realize that same command was also in the test... updated there as well :+1: